components/Header/Header.js

```diff
@@ -1,28 +1,373 @@
+          onClose={this.closeDisclaimerDialog}
+          open={isDisclaimerDialogOpen}
+        >
+          <DialogTitle
+            onClose={this.closeDisclaimerDialog}
+            className={classes.dialogTitle}
+          >
+            AsylumConnect Disclaimer
+            <div
+              className={classes.closeDialogX}
+              onClick={this.closeDisclaimerDialog}
+              onKeyDown={this.closeDisclaimerDialog}
+              role="button"
+              tabIndex="0"
+            >
+              x
+            </div>
+          </DialogTitle>
+          <DialogContent>
+            The AsylumConnect team will do its best to confirm basic facts about
+            service providers listed on this website. However, we lack the
+            necessary resources to fully vet service providers and make no
+            representations regarding the viability or capabilities of any such
+            providers. Consequently, AsylumConnect assumes no responsibility for
+            the actions of providers listed on this website and asylum seekers
+            who contact any such providers do so at their own risk.
+          </DialogContent>
+          <DialogContent className={classes.dialogOkButtonContainer}>
+            <MuiThemeProvider theme={redTheme}>
+              <Fab
+                color="primary"
+                size="small"
+                style={{ width: '100px' }}
+                onClick={this.closeDisclaimerDialog}
+                variant="extended"
+              >
+                Ok
+              </Fab>
+            </MuiThemeProvider>
+          </DialogContent>
+        </Dialog>
+
+        <Dialog
+          onClose={this.closeUserPrivacyStatement}
+          open={isUserPrivacyStatementDialogOpen}
+        >
+          <DialogTitle
+            onClose={this.closeUserPrivacyStatement}
+            className={classes.dialogTitle}
+          >
+            AsylumConnect User Privacy Statement
+            <div
+              className={classes.closeDialogX}
+              onClick={this.closeUserPrivacyStatement}
+              onKeyDown={this.closeUserPrivacyStatement}
+              role="button"
+              tabIndex="0"
+            >
+              x
+            </div>
+          </DialogTitle>
+          <DialogContent>
+            The AsylumConnect catalog uses Google Analytics with&nbsp;
+            <a href="https://support.google.com/analytics/answer/2763052?hl=en">
+              anonymized IP addresses
+            </a>
+            &nbsp;to help analyze how visitors use this site. Google Analytics
+            uses cookies, which are small text files placed on your computer, to
+            collect standard visitor behavior information in an anonymous form.
+            No personally identifiable information is collected about you,
+            unless you explicitly submit that information on this website. If
+            you would like to opt-out of Google Analytics, you may do so by
```

Comment:
```suggestion
            you would like to opt out of Google Analytics, you may do so by
```
(I think "opt out" is a two-word verb here, and someone on the internet agrees with me: https://english.stackexchange.com/questions/385508/opt-out-or-opt-out)